lib/msf/core/exploit/remote/kerberos/client/pkinit.rb

```diff
@@ -1,17 +1,26 @@
 # -*- coding: binary -*-
 
+require 'rasn1'
```

Comment:
is this being used in here?

Reply:
It's required to fix a load order issue when running tests